openlabcmd/openlabcmd/node.py

```diff
@@ -10,17 +10,14 @@ class NodeStatus(object):
 
 class Node(object):
     def __init__(self, name, role, type, ip, heartbeat=None, alarmed=None,
-                 status=None):
+                 status=None, **kwargs):
         self.name = name
         self.role = role
         self.type = type
         self.ip = ip
-        if not heartbeat:
-            self.heartbeat = 0
-        if not alarmed:
-            self.alarmed = False
-        if not status:
-            self.status = NodeStatus.INITIALIZING
+        self.heartbeat = heartbeat if heartbeat else 0
+        self.alarmed = alarmed if alarmed else False
+        self.status = status if status else NodeStatus.INITIALIZING
```

Comment:
Why not set the default values for these parameters, then we can set the them simply?

Reply:
It works here.

But it is not a good custom for python developer IMO. It works terribly for inherit class or dict/list args. Here is an article explain clearly: https://graysonkoonce.com/always-use-none-for-default-args-in-python/

I'm used to using `None` for default value.